src/pages/blog/airship-update-july-2020.md

```diff
@@ -0,0 +1,138 @@
+projects can be shared. On 25 June, Airship presented a project update alongside OpenStack, Kata Containers, StarlingX,
+Zuul, and pilot project OpenInfra Labs.
+
+Catch up on the community meeting with these resources:
+* [_Recording_](https://zoom.us/rec/share/vJF_FqPgxGJJQ9bntR7vaqM7N7i_X6a81yQa8vtcxU06amK9pV9imWJnfHRSUcQ6),
+  Password: 7W!T*i74
+* [_Slides_](https://docs.google.com/presentation/d/16V82OIYfthb3fFlVoes9jZGKMgDIJZ55F8fXqd1M1hU/edit?usp=sharing)
+* [_Agenda_](https://etherpad.opendev.org/p/OSF_Community_Meeting_Q2)
+
+<br>
+
+The project update included:
+
+* The Alpha Milestone, [_completed in May_](https://www.airshipit.org/blog/airship-update-may-2020/)
+* Results of the [_Virtual PTG in June_](https://etherpad.opendev.org/p/airship-virtual-ptg-2020)
+* The newly introduced [_Vulnerability Management Process_](https://docs.airshipit.org/learn/vulnerabilities.html)
+* An update on GitHub Issues utilization
+* Notification of the Technical Committee (TC) and Working Committee (WC) elections
+* Announcement of the [_Airship User Survey_](https://www.surveymonkey.com/r/YKZ9NC2)
+
+<br>
+
+## **WORKING COMMITTEE ELECTION**
+
+The Technical Committee (TC) is excited to announce the 2020-2021 Working Committee (WC) election! This election will
+follow the same two-week cycle that the Technical Committee (TC) election followed last month.
+
+* Nomination Period: 20-26 July
+* Voting Period: 27 July-02 August
+
+<br>
+
+Anyone who has had changes merged within the last 12 months to an Airship project is eligible to run for the Working
+Committee. Anyone who is a Core Reviewer of any Airship Project before the election will be eligible to vote for the WC
+candidates. There are no term limits, but any organization can fill no more than two of the five seats to encourage
+diversity. The Working Committee will meet regularly in an open forum with times and locations published in community
+channels.
+
+For more details concerning eligibility, responsibilities, and election processes, please consult the
+[_Airship Governance_](https://opendev.org/airship/governance).
+
+<br>
+
+## **AIRSHIP USER SURVEY**
+
+If you are evaluating or running Airship, share your feedback in the [_Airship User Survey_](
+https://www.surveymonkey.com/r/YKZ9NC2)! This survey provides users an opportunity to provide anonymous feedback to
+influence the community and software direction. By sharing information about your configuration and requirements, the
+Airship Working Committee will advocate on your behalf. We will present the 2020 survey results at the Fall Summit in
+November. Take the [_Airship User Survey_](https://www.surveymonkey.com/r/YKZ9NC2) to be included in this round of
+analysis.
+
+<br>
+
+## **AIRSHIP 2.0 PROGRESS**
+
+The progress shown below is for [_airshipctl_](https://opendev.org/airship/airshipctl), the new Airship 2.0 client.
```

Comment:
Do we want to talk about Airship UI here or just Airshipctl?

Reply:
Conflicted on UI, there's only a handful of issues for each milestone https://github.com/airshipit/airshipui/milestones and we're past due on what Alpha was supposed to be.  I think until UI gets better scoped we leave it out.